apps/app/src/app/shared/input-error/input-error.component.ts

```diff
@@ -1,11 +1,11 @@
-import { NgFor, NgIf } from '@angular/common';
+import { NgFor } from '@angular/common';
 import { Component, computed } from '@angular/core';
 import { HlmInputErrorDirective } from '@spartan-ng/ui-input-helm';
 import { injectErrorField } from 'ng-signal-forms';
 
 @Component({
 	selector: 'spartan-input-error',
-	imports: [NgIf, NgFor, HlmInputErrorDirective],
+	imports: [NgFor],
```

Comment:
super nit 2: I distinctly remember discussing the migration to \@for & \@if on Discord or somewhere - did we not get around to that? Again, you need not pick that up with this PR - whatever for works!